project1.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="project1.css" type="text/css">
+  <link href="https://font.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <title>Setup and Mobile Version Skeleton</title>
+</head>
+
+<body>
+  <nav class="navbar">
+    <div class="container">
+    <div class="logo">My Logo</div>
+  
+  <span class="material-symbols-outlined">
+menu
+</span>
+</div>
+</nav>
+
+<header>
+<div class="container-1">
+<div class="background-image"> </div>
+<h1>I'm Mariane Glad to see you!</h1>
+
+  <p>I'm a software developer! I can help you build a product, feature or website
+    look through some of my work and experience! If you like what you see and have
+    a project you need coded, don't hesitate to contact me.</p>
+
+
+    <h5>LET'S CONNECT</h5>
+
+    <a><img src="socialmediaicons.jpg" alt=" "></a>
```

Comment:
- [ ] the social media icons should be an unordered list `<ul>` and every icon should be a list item `<li>` it will look something like in the below example:
```
<ul>
  <li><a>icon</a></li>
  <li><a>icon</a></li>
  <li><a>icon</a></li>
  ...
</ul>
```